security/security-design/fn-datasafe-dbaudit-to-oci-logging/terraform/provider.tf

```diff
@@ -0,0 +1,61 @@
+################################################################################
+#Copyright (c) 2023 Oracle and/or its affiliates.
+# 
```

Comment:
Can be shorter.
\# Copyright (c) 2023, Oracle and/or its affiliates. All rights reserved.
\# This software is dual-licensed to you under the Universal Permissive License (UPL) 1.0 as shown at https://oss.oracle.com/licenses/upl or Apache License 2.0 as shown at http://www.apache.org/licenses/LICENSE-2.0. You may choose either license.